### DISCUSSION TITLE
How can I restore a specific subfolder from a snapshot?

### DISCUSSION THREAD

**Discussion by jeremy-lockwood** [cupcakearmy/autorestic#356] [Q&A]
Restic's documentation for this says to use `<snapshot>:<subfolder>` syntax, but this just results in the following error: 
```
invalid id "latest:/home/user/app": no matching ID found for prefix "latest:/home/user/app"
Error: exit status 1
```
basically, i want to restore the CONTENTS of /home/user/app in my snapshot to /home/user/app on my system. it'd be really cool if i could define this in the .autorestic.yml file, too.

thanks!

**Comment by miguelarios** [comment]
You should use the `exec` command and just follow what the restic documentation says. I just did it to test a restore for a file.

`autorestic --config /path/to/config.yml exec --backend local restore -- <snapshot_id> --target ~/Documents/test --include "/path/to/file/picture.jpg"`

**Reply by Thanoonazad** [reply]
How is it possible to add the --include into the yaml file? As far as I can see, there is no option for the restore.
